src/Simplex/Messaging/Agent/Store/SQLite/Schema.hs

```diff
@@ -7,78 +7,99 @@ module Simplex.Messaging.Agent.Store.SQLite.Schema where
 import Database.SQLite.Simple
 import Database.SQLite.Simple.QQ (sql)
 
+-- service_name is either a service name or a port number, see Network.Socket.Info.ServiceName
 servers :: Query
 servers =
   [sql|
-    CREATE TABLE IF NOT EXISTS servers
-      ( server_id INTEGER PRIMARY KEY,
-        host TEXT NOT NULL,
-        port TEXT,
-        key_hash BLOB,
-        UNIQUE (host, port)
-      )
+    CREATE TABLE IF NOT EXISTS servers(
+      host TEXT NOT NULL,
+      service_name TEXT NOT NULL,
```

Comment:
service_name can actually be NULL. We should not be defaulting it to 5223, in the future we will support hostnames with unspecified ports - they idea is to be able to fallback to say web-sockets if 5223 is not opened.

Reply:
even though SQLite wrongfully supports it (due to a bug in early versions), agreed to use empty strings / underscores and convert it to `Nothing` instead of NULLs in primary key

Reply:
another option would be to keep host & service_name in a single field, would require addition parsing though